api/api/services/features.py

```diff
@@ -104,7 +101,7 @@ async def _is_company_email_domain(user_email_domain: str | None) -> bool:
 
     @staticmethod
     async def get_feature_sections_preview(user_domain: str | None = None) -> list[FeatureSectionPreview]:
-        is_company_email_domain = await FeatureService._is_company_email_domain(user_domain)
+        show_company_section = user_domain is None or await FeatureService._is_company_email_domain(user_domain)
```

Comment:
maybe add a comment of why we are showing the company section for anonymous user ?

Reply:
https://github.com/WorkflowAI/WorkflowAI/pull/32/commits/604d3eedffc3b99559f99008703dc081d095fc77